airbyte-integrations/connectors/source-linkedin-ads/manifest.yaml

```diff
@@ -1660,6 +1660,21 @@ dynamic_streams:
               "value",
             ]
           value: "{{components_values['pivot_by']}}"
+
+# LinkedIn Ads does not publish their standard API rate limits.
+# https://learn.microsoft.com/en-us/linkedin/shared/api-guide/concepts/rate-limits
+# I looked into our test account to establish some baselines, but it seems like each endpoint
+# has its own individually tracked rate limit that resets daily. There are also tiers, but
+# there is not a good way of ascertaining what value are for each. There are 19 streams so
+# as a standard default will be set to 19 so each stream can at least run in parallel. This
+# could require some tuning.
+#
+# It is likely that certain customers may have a greater limit provisioned so allow for a max of 50.
+concurrency_level:
+  type: ConcurrencyLevel
+  default_concurrency: "19"
```

Comment:
I decided to leave this as non-configurable for user at this moment given the above. We can explore making this user-configurable later, but its easier to open this up later while still getting a nice performance bump